modules/consensus/accept_test.go

```diff
@@ -439,16 +450,19 @@ func (cst *consensusSetTester) testSpendSiacoinsBlock() error {
 
 	// Find the destAddr among the outputs.
 	var found bool
-	for id, output := range cst.cs.siacoinOutputs {
+	cst.cs.db.forEachSiacoinOutputs(func(id types.SiacoinOutputID, output types.SiacoinOutput) {
 		if id == outputID {
 			if found {
-				return errors.New("output found twice")
+				err = errors.New("output found twice")
 			}
 			if output.Value.Cmp(txnValue) != 0 {
-				return errors.New("output has wrong value")
+				err = errors.New("output has wrong value")
```

Comment:
You want to let it finish iterating even though you are going to return an error at the end anyway?